content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,222 @@
+you build a native mobile app for iPhone. A colleague argues that a responsive
+web app is the only way to go. Who's right? Are there alternative options? The
+decision of how to build your app is quite important -- affecting project scope,
+timeline, and budget -- but understanding when to choose one option over another
+can be overwhelming.
+
+The great news is that you don't need to spend years becoming a web developer
+yourself in order to make a wise choice. In the following three-part series,
+we'll dig into the three main options for building mobile apps: Responsive Web
+Apps, Progressive Web Apps, and Native Mobile Apps, and how to determine the
+best option for your project. As you read, think about who your app will serve,
+what their most pressing needs are, and how your app will meet those needs.
+
+**Read all three parts to compare app types:**
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/)
+
+## What are Responsive Web Apps, PWAs, and Native Mobile Apps?
+
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** -- An application that runs in web browsers, such
+  as Chrome, Firefox, or Safari, and responds to fit any screen size on any
+  device.
+- **Progressive Web App (PWA)** -- A responsive web app with additional
+  capabilities that mimic a native mobile app.
+- **Native Mobile App** -- An app built specifically for one particular mobile
+  operating system, like iOS or Android.
+
+{{ embed.img(
+  src='blog/2023/comparison-chart.jpg',
+  alt='A comparison chart showing the differences between
+    responsive web apps, progressive web apps, and native
+    mobile apps'
+) }}
+
+## Responsive Web Apps -- Best for Wide Reach
+
+> **TL;DR**
+>
+> If your top priority is to reach as wide a range of
+> people as possible, as quickly as possible, while keeping
+> development and maintenance costs down, a responsive web app is
+> likely the best option for building your digital product.
+
+An RWA is essentially a website that resizes and rearranges its content --
+responding to fit whatever screen size a person chooses to use. RWAs are built
+using standard web technologies like HTML, CSS, and JavaScript. They are
+accessible from any device with a web browser such as Chrome, Firefox, Microsoft
+Edge, Safari, or a mobile browser.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/yes.png',
+    alt: 'The word "yes" with a smiley and sparkles'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Choose an RWA when…
+
+### Lower Cost
+*…when it's important to keep initial development costs low.*
+
+Responsive web apps use a single codebase. The term *codebase* is
+developer-speak for the collection of code used to build an application.
```

Comment:
I think a footnote could be a good idea. I'd actually like to go in the direction of defining more terms because I'd like people who are unfamiliar with apps to feel very comfortable reading this article.

Reply:
@SondraE That makes sense to me. You can look at Mia's recent `zero-units` article for how to add footnotes: https://github.com/oddbird/oddleventy/blob/main/content/blog/2022/zero-units.md?plain=1#L174-L185

Reply:
I moved this sentence into a footnote: https://github.com/oddbird/oddleventy/commit/23eec2e9cd9ac4bd36504274c3d6fe4b6e2bfb30